virttest/lvm.py

```diff
@@ -555,12 +558,13 @@ def setup_lv(self):
         """
         lv_name = self.params["lv_name"]
         lv_size = self.params["lv_size"]
+        lv_extra_options = self.params["lv_extra_options"]
```

Comment:
Sorry, this is not correct, should be  `lv_extra_options = self.params.get("lv_extra_options")`, since for other scenarios which do not need to set `lv_extra_options` will raise an error.